src/styles/blocks/product.scss

```
@@ -0,0 +1,69 @@
+.product {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    align-items: center;
+    padding-left: 3px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--light-text-color);
+    margin-bottom: 7px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    color: var(--light-text-color);
+  }
+
+  &__price-dollar {
+    font-size: 16px;
+    font-weight: 700;
+  }
```

No need to duplicate selectors
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    align-items: center;
    
    &-text {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```